base_external_dbsource_sqlite/demo/base_external_dbsource.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record model="base.external.dbsource" id="demo_sqlite">
+        <field name="name">SQLite Demo</field>
+        <field name="conn_string">Data Source=:memory:;Version=3;New=True;</field>
+        <field name="password">password</field>
```

Comment:
This doesn't work out of the box. Please use a better example:

```suggestion
        <field name="conn_string">sqlite://</field>
        <field name="password"></field>
```

Docs: https://docs.sqlalchemy.org/en/20/dialects/sqlite.html#using-a-memory-database-in-multiple-threads

Reply:
I committed this myself.